test/integration/globals/app-nav.js

```diff
@@ -431,7 +472,7 @@ context('App Nav', function() {
       })
       .routeActions()
       .routePatient()
-      .visit();
+      .visit('/', { isRoot: true });
```

Comment:
This seems to have fixed the `404`ing issue. At least it did on my local machine 🤞